server/routes/index.js

```diff
@@ -0,0 +1,89 @@
+import User from '../controllers/usersController';
+import Recipe from '../controllers/recipesController';
+import Review from '../controllers/reviewsController';
+import Vote from '../controllers/votesController';
+import Favourite from '../controllers/favouritesControllers';
+import getToken from '../middlewares/getToken';
+import verifyToken from '../middlewares/verifyToken';
+
+
+const routes = (app) => {
+  app.get('/', (req, res) => {
+    res.status(200)
+      .send('Welcome to more-recipes api');
+  });
+
+  // create a user
+  app.post('/api/v1/users/signup', User.createUser);
+
+  // user signs in
+  app.post('/api/v1/users/signin', User.userLogin);
+
+  // an auth user can view his/her profile
+  app.get(
+    '/api/v1/users/profile',
+    getToken, verifyToken, User.getUserProfile
+  );
+
+  // auth user can update their profile
+  app.put(
```

Comment:
Group your routes to maintain a DRY/Lean code.
```
app.get(
  '/api/v1/users/profile',
  getToken, verifyToken, User.getUserProfile
);

app.put(
  '/api/v1/users/profile',
  getToken, verifyToken, User.updateUserProfile
);
```

Since both `get` and `put` are on the same route, we can chain them like so:
```
app.route('/api/v1/users/profile')
  .get(getToken, verifyToken, User.getUserProfile)
  .put(getToken, verifyToken, User.updateUserProfile);
```

Reply:
Fixed